msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -21,6 +21,16 @@ Resource-specific consent (RSC) is a Microsoft Teams and Microsoft Graph API int
 
 The granular, Teams-specific, RSC permissions define what an application can do within a specific resource.
 
+The following are the links for Resource specific permissions of older version(placeholder for version):
+
+* [Resource-specific permissions for a team](#resource-specific-permissions-for-a-team)
+* [Resource-specific permissions for a chat](#resource-specific-permissions-for-a-chat)
+
+The following are the Resource specific permissions of latest version(placeholder for version):
+
+* [RSC Delegated permission](#rsc-delegated-permission)
```

Comment:
This is not what I meant. 

This is a Microsoft Graph specific document. Delegated RSC permissions have no relevance in MS Graph today. These should not be mentioned at all in this document.

Reply:
Hi @AkJo , removed the section as suggested.